docs/types/power_types/modify_enchantment_level.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Modify Enchantment Level (Power Type)
+date: 2023-10-09
+---
+
+# Modify Enchantment Level
+
+[Power Type](../power_types.md)
+
+Applies/modifies the level of the specified enchantment to/from the entity.
+
+Type ID: `origins:modify_enchantment_level`
+
+!!! warning
+
+    This power type will only work if an item is equipped.
```

Comment:
```suggestion
```
This note should be removed since the power type will apply (instead of modify) the specified enchantment if no item that has the specified enchantment is equipped. The modifiers, in this case, will have a base value of `0`.